src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -279,6 +289,20 @@ public static void FindNatvisInSolution(NatvisLoader loader)
     }
 
     /// <summary>
+    /// Provides interactions with the host's debugger
+    /// </summary>
+    public static class HostDebugger
+    {
+        /// <summary>
+        /// Attach to a process using the provided options
```

Comment:
How about - asks the host to async spin up a new instance of the debug engine and go through the launch sequence using the specified options.

Do we want to also maybe call this 'StartDebugChildProcess' or something like that?